x/auth/client/cli/tx_sign.go

```diff
@@ -89,6 +92,12 @@ func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
 		}
 		scanner := authclient.NewBatchScanner(txCfg, infile)
 
+		if !clientCtx.Offline {
+			txFactory = txFactory.WithAccountNumber(0).WithSequence(0)
+		} else {
+			txFactory = txFactory.WithAccountNumber(txFactory.AccountNumber()).WithSequence(txFactory.Sequence())
```

Comment:
Why is this needed? Setting the txFactory from values of txFactory?

Reply:
we don't need to. Thanks @atheeshp